src/background.js

```diff
@@ -16,11 +16,22 @@ function log(...args) {
   if (DEBUG) { console.log('SnoozeTabs (BE):', ...args); }  // eslint-disable-line no-console
 }
 
+var iconData;
```

Comment:
`let` is the new `var`.
Not sure if we want to add `no-var` and `prefer-const` to the ESLint config.